pkg/controller/jobframework/setup.go

```diff
@@ -83,6 +83,7 @@ func SetupControllers(mgr ctrl.Manager, log logr.Logger, opts ...Option) error {
 				if err = cb.SetupWebhook(mgr, opts...); err != nil {
 					return fmt.Errorf("%s: unable to create webhook: %w", fwkNamePrefix, err)
 				}
+				EnableIntegration(name)
```

Comment:
If we forgot to add this line all tests would pass, but the code would not work from the user perspective. 
Would it be feasible to call `SetupControllers` from the integration tests to cover this (potentially in a follow up if requires refactoring)?

Reply:
> If we forgot to add this line all tests would pass, but the code would not work from the user perspective.

I don't think I follow.

> Would it be feasible to call `SetupControllers` from the integration tests to cover this (potentially in a follow up if requires refactoring)?

Not really, in the integration test we need more flexibility (start only the controllers and not the web-hook server, only consider `rayjob` integration enabled but run the `raycluster`).

Reply:
Added a check in the setup unit test.